plugins/inputs/opcua/opcua_util.go

```diff
@@ -146,7 +146,7 @@ func pemBlockForKey(priv interface{}) *pem.Block {
 
 // OPT FUNCTIONS
 
-func generateClientOpts(endpoints []*ua.EndpointDescription, certFile, keyFile, policy, mode, auth, username, password string, requestTimeout time.Duration) []opcua.Option {
+func generateClientOpts(endpoints []*ua.EndpointDescription, certFile, keyFile, policy, mode, auth, username, password string, requestTimeout time.Duration) ([]opcua.Option, error) {
```

Comment:
You can switch off the linter for that one line something like:

//revive:disable-next-line:argument-limit

Reply:
sweet thanks